sendgrid/sendgrid.py

```diff
@@ -58,3 +57,14 @@ def api_key(self, value):
     @property
     def impersonate_subuser(self):
         return self._impersonate_subuser
+
+
+class SendGridClient(SendGridAPIClient):

```

Comment:
Sure! We can add it to the `SendGridAPIClient` class instead.
On the other hand, if we remove `SendGridClient`, how would we catch the exceptions?  `sg.client` is a object of class `python_http_client.Client`. So when we do `sg.client.mail.send.post(...)`, we have to catch a HTTPError in the end application. I'm not sure if we can't handle this exception in `SendGridAPIClient` class.